seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/java/org/apache/seatunnel/e2e/connector/file/local/LocalFileIT.java

```diff
@@ -149,6 +151,13 @@ public class LocalFileIT extends TestSuiteBase {
                         "/seatunnel/read/tar_gz/txt/multifile/multiTarGz.tar.gz",
                         container);
 
+                Path txtGz =
```

Comment:
I briefly looked at the error report. It may be because the files in the container are messed up. You can adjust the path of the files in the container.

Reply:
The file path in the container has been adjusted, but the case still cannot pass. Please help me take a look

Reply:
At night or at noon tomorrow